app/controllers/api/v1/yelp_search_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::YelpSearchController < ApplicationController
+  def index
+    results = YelpFacade.search_nearby(params[:cocktail_name], params[:location])
```

Comment:
Will we need a few conditionals for sad paths? Maybe a refactor for later...

Reply:
oh my gosh, I totally forgot to sad path.

Reply:
no worries. we can merge this and you can build the sad paths. i forgot as well for the other stories.